src/main/apisix/Chart.yaml

```diff
@@ -0,0 +1,46 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: v2
+description: Apache APISIX is a dynamic, real-time, high-performance API gateway, based on the Nginx library and etcd.
+icon: https://dashboard.apisix.dev/favicon.png
+maintainers:
+- email: liling@apache.org
+  name: liling
+name: apisix
+dependencies:
+- name: etcd
+  condition: etcd.enabled
```

Comment:
I submitted etcd to `test` repo and we can declare it as follows:

```
dependencies:
- name: etcd
  version: 0.1.0
  repository: https://charts.kubesphere.io/test
  condition: etcd.enabled
```

Reply:
so, do I need to delete those etcd charts files in the `main` repo?

Reply:
yes, please remove etcd dir.

Reply:
done